catalog-spec/catalog-spec.md

```diff
@@ -85,13 +85,13 @@ For a full discussion of the situations where relative and absolute links are re
-| child  | URL to a child STAC entity (Catalog or Collection).                                                                                                                                                                                                             |
-| item   | URL to a STAC Item.                                                                                                                                                                                                                                             |
+| Type   | Description                                                                                                                                                                                                                                                     | Media Type                                 |
+| ------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------ |
+| self   | STRONGLY RECOMMENDED. *Absolute* URL to the location that the Catalog file can be found online, if available. This is particularly useful when in a download package that includes metadata, so that the downstream user can know where the data has come from. | application/json                           |
+| root   | STRONGLY RECOMMENDED. URL to the root STAC Catalog or [Collection](../collection-spec/README.md). Catalogs should include a link to their root, even if it's the root and points to itself.                                                                     | application/json                           |
+| parent | URL to the parent STAC entity (Catalog or Collection). Non-root Catalogs should include a link to their parent.                                                                                                                                                 | application/json                           |
+| child  | URL to a child STAC entity (Catalog or Collection).                                                                                                                                                                                                             | application/json                           |
+| item   | URL to a STAC Item.                                                                                                                                                                                                                                             | application/geo+json (or application/json) |
```

Comment:
```suggestion
| item   | URL to a STAC Item.                                                                                                                                                                                                                                             | application/geo+json (preferred) or application/json |
```

It might help to indicate that `application/geo+json` should be used if possible ... the original wording made it seems as though they were interchangeable.